src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -447,7 +447,7 @@ public function getProtocolVersion()
      * If the status text is null it will be automatically populated for the known
      * status codes and left empty otherwise.
      *
-     * @return Response
+     * @return $this
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1047696/19653642/8fbd56ce-9a15-11e6-8420-4ae5c4241213.png)

Changing `FormConfigBuilderInterface::addEventListener` from `@return self` to `@return $this`

![image](https://cloud.githubusercontent.com/assets/1047696/19653688/bb4990be-9a15-11e6-9104-7e06e7a3e780.png)